go.sum

```diff
@@ -386,8 +386,6 @@ github.com/denisenkom/go-mssqldb v0.10.0/go.mod h1:xbL0rPBG9cCiLr28tMa8zpbdarY27
 github.com/denverdino/aliyungo v0.0.0-20190125010748-a747050bb1ba/go.mod h1:dV8lFg6daOBZbT6/BDGIz6Y3WFGn8juu6G+CQ6LHtl0=
 github.com/dgrijalva/jwt-go v0.0.0-20170104182250-a601269ab70c/go.mod h1:E3ru+11k8xSBh+hMPgOLZmtrrCbhqsmaPHjLKYnJCaQ=
 github.com/dgrijalva/jwt-go v3.2.0+incompatible/go.mod h1:E3ru+11k8xSBh+hMPgOLZmtrrCbhqsmaPHjLKYnJCaQ=
```

Comment:
> Finally. I tried my best, but I just cannot remove github.com/dgrijalva/jwt-go from `go.sum`. I don't know what's referencing it. Pointers welcome

It is required by 2 modules:
* `github.com/influxdata/influxdb`@`v1.9.5`, a direct dependency
* `github.com/Azure/go-autorest/autorest/adal`, required by `github.com/golang-migrate/migrate`, required at https://github.com/apache/trafficcontrol/blob/33a1ee0f30ccb0b927105e2a96187883a0a3b6d8/traffic_ops/app/db/admin.go#L36

Since `github.com/dgrijalva/jwt-go` is no longer a vendored dependency, I don't see a problem with these lines staying in `go.sum`, from ATC's perspective.